packages/discord-bot/src/commands/help.ts

```diff
@@ -0,0 +1,39 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import logger from 'jet-logger';
+import { helpHandler } from '../handlers/help';
+import { HelpCommandBuilder, Command } from '../interfaces/Command';
+
+export const help: HelpCommandBuilder = (commands) => {
+  const commandNames: [name: string, value: string][] = Array.from(
+    commands
+  ).map((i) => [i[0], i[0]]);
+  return {
+    help: {
+      data: new SlashCommandBuilder()
+        .setName('help')
+        .setDescription('Shows help text for praise')
+        .addStringOption((option) =>
+          option
+            .addChoices(commandNames)
+            .setName('cmd')
```

Comment:
Should this be "command" instead of "cmd"? We're not short on space or anything.